src/vs/workbench/contrib/preferences/browser/settingsEditor2.ts

```diff
@@ -1210,10 +1243,47 @@ export class SettingsEditor2 extends EditorPane {
 			this.hasWarnedMissingSettings = true;
 		}
 
-		const commonlyUsed = resolveSettingsTree(commonlyUsedData, dividedGroups.core, this.logService);
-		resolvedSettingsRoot.children!.unshift(commonlyUsed.tree);
+		const toggleData = await getExperimentalExtensionToggleData(this.workbenchAssignmentService, this.environmentService, this.productService);
+		if (toggleData) {
+			for (const key in toggleData.settingsEditorRecommendedExtensions) {
+				const prerelease = toggleData.settingsEditorRecommendedExtensions[key].onSettingsEditorOpen!.prerelease;
+				const groupTitle = toggleData.settingsEditorRecommendedExtensions[key].onSettingsEditorOpen!.groupTitle;
+				const extensionId = (typeof prerelease === 'string' && this.productService.quality !== 'stable') ? prerelease : key;
+				let extension: IGalleryExtension | Readonly<IRelaxedExtensionDescription> | undefined = await this.extensionService.getExtension(extensionId);
+				if (!extension) {
+					const galleryExtensions = await this.extensionGalleryService.getExtensions([{ id: extensionId }], CancellationToken.None);
+					extension = galleryExtensions.find(extension => extension.identifier.id.toLowerCase() === extensionId.toLowerCase());
+					if (!extension) {
+						continue;
+					}
+				}
+				const extensionName = extension?.displayName ?? extension?.name ?? extensionId;
+				const settingKey = `${key}.manageExtension`;
+				const toggleSetting: ISetting = {
+					range: nullRange,
+					key: settingKey,
+					keyRange: nullRange,
+					value: null,
+					valueRange: nullRange,
+					description: [extension?.description || ''],
+					descriptionIsMarkdown: false,
+					descriptionRanges: [],
+					title: localize('manageExtension', "Manage {0}", extensionName),
+					scope: ConfigurationScope.APPLICATION,
```

Comment:
May be scope can be made window? Making this application scope shows that it is applied for All profiles and I think it is irrelevant here.